UiPathTeam.Extensions/UiPathTeam.Extensions.Activities/Properties/Resources.resx

```diff
@@ -465,4 +465,72 @@
     <value>List of values in the dictionary</value>
     <comment>property description</comment>
   </data>
-</root>
+  <data name="Authentication_Category" xml:space="preserve">
+    <value>Authentication</value>
+    <comment>category</comment>
+  </data>
+  <data name="Common_Category" xml:space="preserve">
+    <value>Common</value>
+    <comment>category</comment>
+  </data>
+  <data name="Input_Category" xml:space="preserve">
+    <value>Input</value>
+    <comment>category</comment>
+  </data>
+  <data name="Output_Category" xml:space="preserve">
+    <value>Output</value>
+    <comment>category</comment>
+  </data>
+  <data name="Options_Category" xml:space="preserve">
+    <value>Options</value>
+    <comment>category</comment>
+  </data>
+  <data name="ValidationScope_Error" xml:space="preserve">
+    <value>Cannot be used outside of a {0}.</value>
+    <comment>error message; {0} is a container activity (e.g. a scope)</comment>
+  </data>
+  <data name="ValidationValue_Error" xml:space="preserve">
+    <value>Please provide a value for {0}.</value>
+    <comment>error message; {0} is the name of a missing property</comment>
+  </data>
+  <data name="ValidationExclusiveProperties_Error" xml:space="preserve">
+    <value>Please provide a value either for {0} or {1}, but not both.</value>
+    <comment>error message; {0} &amp; {1} are mutually exclusive activities</comment>
+  </data>
+  <data name="ValidationOverloadGroup_Error" xml:space="preserve">
+    <value>Please provide values for both {0} and {1}.</value>
+    <comment>error message; {0} &amp; {1} are complementary properties</comment>
+  </data>
+  <data name="GetDictionaryKeys_DisplayName" xml:space="preserve">
+    <value>Get Dictionary Keys</value>
+    <comment>activity name</comment>
+  </data>
+  <data name="GetDictionaryKeys_Description" xml:space="preserve">
+    <value>Get a list of the keys in the dictionary.</value>
+    <comment>activity description</comment>
+  </data>
+  <data name="GetDictionaryKeys_In_Dictionary_DisplayName" xml:space="preserve">
+    <value>in_Dictionary</value>
+    <comment>property name</comment>
+  </data>
+  <data name="GetDictionaryKeys_In_Dictionary_Description" xml:space="preserve">
+    <value>The dictionary you want to retrieve the keys from.</value>
+    <comment>property description</comment>
+  </data>
+  <data name="Input_Category" xml:space="preserve">
+    <value>Input</value>
+    <comment>category</comment>
+  </data>
```

Comment:
remove duplicative keys from the wizard

Reply:
Removed 520-523 Input_Category duplicate key